HTML/Javascript.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+
+<body style="font-family: Vemana2000, sans-serif;">
+<ul style="list-style-type: none; overflow: hidden; background-color: black; margin-left: 0%; padding-top:25px; padding-bottom: 0px;">
+    <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/99/Unofficial_JavaScript_logo_2.svg/2048px-Unofficial_JavaScript_logo_2.svg.png" width="40px;">
+    </li>
+    <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Git.html" style="color:white; text-decoration: none;">Git</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; ">
+        <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+    </li>
+</ul>
+<div style="margin-left: 50px">
+<h2 > Javascript</h2>
```

Comment:
:hammer_and_wrench: Leerzeichen zu viel
```suggestion
<h2> Javascript</h2>
```